UnitedIncome/components
# Allows for only a single item to be open at once on the question answer component

### Description
> Please provide a brief description of your changes below.

Allows for only a single item at once to be open on the QuestionAnswer component.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Test the question answer component.
* Test the existing expand/collapse.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
